src/components/panel/panel.spec.js

```diff
@@ -868,6 +895,87 @@ describe('$mdPanel', function() {
       });
     });
 
+    describe('CSS class logic:', function() {
+      it('should add a class to the container/wrapper when toElement=false',
+        function() {
+          openPanel(DEFAULT_CONFIG);
+
+          panelRef.addClass('my-class');
+
+          expect(PANEL_WRAPPER_CLASS).toHaveClass('my-class');
+          expect(PANEL_EL).not.toHaveClass('my-class');
+        });
+
+      it('should add a class to the element when toElement=true', function() {
+        openPanel(DEFAULT_CONFIG);
+
+        panelRef.addClass('my-class', true);
+
+        expect(PANEL_EL).toHaveClass('my-class');
+        expect(PANEL_WRAPPER_CLASS).not.toHaveClass('my-class');
+      });
+
+      it('should remove a class from the container/wrapper when fromElement=false',
+        function() {
+          openPanel(DEFAULT_CONFIG);
+
+          panelRef.addClass('my-class');
+
+          expect(PANEL_WRAPPER_CLASS).toHaveClass('my-class');
```

Comment:
super nit: Can you keep the order of these the same for all the tests in this describe? Either put `PANEL_WRAPPER_EL` expect first or `PANEL_EL` first, but don't swap them.